app/ide-desktop/lib/server/src/index.mjs

```diff
@@ -17,10 +17,16 @@ export async function start({ root, assets, port }) {
 
     const freePort = await portfinder.getPortPromise({ port: port ?? DEFAULT_PORT })
 
+    // Added to test if it fixes the issue: https://github.com/expressjs/serve-static/issues/155
+    // It does not, should probably be removed. To be checked when the issue is resolved.
+    const setHeaders = (res) => {
+        res.setHeader('Cache-Control', 'no-store')
+    }
```

Comment:
1. Why is this needed?
2. Do you have any docs explaining these options? I'd love to understand them + add comments to the code.

Reply:
To have more accurate profiling info from our profiler. This was the outcome of a long discussion https://discord.com/channels/401396655599124480/1052238356311986269 In this thread there are also links explaining those headers.

Reply:
Wait, so the headers are not set now? I remember you've been working on that topic, so probably you changed some code, right? I'm asking to avoid code duplication.

Reply:
I did it on my very specialized branch where I added tons of profiles for the component browser. I forgot to make a separate PR with those headers only. That why I asked you to take the occasion and add those headers now